maplayer.c

```diff
@@ -1604,9 +1604,19 @@ char* msLayerBuildSQLOrderBy(layerObj *layer)
   if( layer->sortBy.nProperties > 0 ) {
     int i;
     for(i=0;i<layer->sortBy.nProperties;i++) {
-      char* escaped = msSmallMalloc(strlen(layer->sortBy.properties[i].item) + 3);
-      sprintf(escaped, "\"%s\"", layer->sortBy.properties[i].item);
-      if( i > 0 )
+      char* escaped = msLayerEscapePropertyName(layer, layer->sortBy.properties[i].item);
+      //Enclose property name in double quotes (if it isn't yet) to ensure that mixed case property names are supported
+      if (escaped[0] != '"')
```

Comment:
Tried this (easy fix) and it works, tx for the suggestion. 
I was wondering why there was the special version of msOracleSpatialEscapePropertyName. I'm guessing that some developers were interested in using the "lazy" Oracle naming where `COLUMN` and `column` and `Column` all mean the same as Oracle internally translates all non-double-quoted names to uppercase, only `"column"` really means the lower case variant.
As the side effect of the change users will now be unable to use `column` for an (e.g. WFS) parameter name when in Oracle it's `COLUMN`.